gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/api-tab-subscriptions.component.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { AsyncPipe } from '@angular/common';
+import { Component, inject, Input, OnInit } from '@angular/core';
+import { catchError, Observable } from 'rxjs';
+import { of } from 'rxjs/internal/observable/of';
+
+import { Subscription } from '../../../entities/subscription/subscription';
+import { SubscriptionService } from '../../../services/subscription.service';
+
+@Component({
+  selector: 'app-api-tab-subscriptions',
+  standalone: true,
+  imports: [AsyncPipe],
+  templateUrl: './api-tab-subscriptions.component.html',
+  styleUrl: './api-tab-subscriptions.component.scss',
+})
+export class ApiTabSubscriptionsComponent implements OnInit {
+  @Input()
+  apiId!: string;
+
+  subscriptionsList$!: Observable<Subscription>;
+  private subscriptionService = inject(SubscriptionService);
+
+  constructor() {}
+
+  ngOnInit(): void {
+    this.subscriptionsList$ = this.loadSubscriptions$();
+  }
+
+  private loadSubscriptions$(): Observable<Subscription> {
+    return this.subscriptionService.list(this.apiId).pipe(catchError(_ => of()));
```

Comment:
What about an empty list in `catchError` so that the message for no subscriptions is shown?

Reply:
Done